diff -up python-cgen-2020.1/doc/source/conf.py.omv~ python-cgen-2020.1/doc/source/conf.py
--- python-cgen-2020.1/doc/source/conf.py.omv~	2024-07-07 18:54:09.780015670 +0200
+++ python-cgen-2020.1/doc/source/conf.py	2024-07-07 19:05:15.714903651 +0200
@@ -54,11 +54,9 @@ copyright = u'2011, Andreas Kloeckner'
 # built documents.
 #
 # The short X.Y version.
-import re
-ver_re = re.compile(r'version\s*=\s*"([0-9a-z.]+)"')
-version = [ver_re.search(line).group(1) 
-        for line in open("../../setup.py").readlines() 
-        if ver_re.search(line)][0]
+sys.path.insert(0, '../../cgen')
+import version as V
+version=''.join([str(x) + '.' for x in V.VERSION])[:-1]
 # The full version, including alpha/beta/rc tags.
 release = version
 
@@ -195,8 +193,8 @@ htmlhelp_basename = 'cgendoc'
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, document class [howto/manual]).
 latex_documents = [
-  ('index', 'cgen.tex', ur'cgen Documentation',
-   ur'Andreas Kloeckner', 'manual'),
+  ('index', 'cgen.tex', 'cgen Documentation',
+   'Andreas Kloeckner', 'manual'),
 ]
 
 # The name of an image file (relative to this directory) to place at the top of
